pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3621,15 +3621,14 @@ protected OffloadProcessStatus internalOffloadStatus(boolean authoritative) {
                     validateAdminAccessForTenant(pulsar,
                             clientAppId, originalPrincipal, topicName.getTenant(), authenticationData);
                 } catch (RestException authException) {
-                    log.warn("Failed to authorize {} on cluster {}", clientAppId, topicName.toString());
+                    log.warn("Failed to authorize {} on topic {}", clientAppId, topicName, authException);
```

Comment:
IMHO, we don't need log stack trace here. `authException` can be set as the cause of the following PulsarClientException.